tests/unit/button-plain.spec.js

```diff
@@ -0,0 +1,21 @@
+import { shallowMount } from '@vue/test-utils';
+import ButtonPlain from '../../src/popup/router/components/ButtonPlain.vue';
+
+describe('ButtonPlain', () => {
+  it('should emit click event on click', async () => {
```

Comment:
IMO, I don't think tha we should test that button is emiting click event, every other test is using button, and if the click would fail the test would fail as well.